source/developer/contribution-guide.md

```diff
@@ -66,6 +66,8 @@ After submitting a PR, before it is merged:
 3. Dev review must pass
     - Two core committers will review the pull request and either give feedback or approve the PR.
     - Any comments will need to be addressed before the pull request is ready to merge.
+4. CSS review must pass
```

Comment:
Hmm good question.

I think that can happen during the "PM review" stage, so it can be added in this section: https://github.com/mattermost/platform/blob/master/CONTRIBUTING.md#stage-1-pm-review